tests/rt.conf

```diff
@@ -197,3 +191,13 @@ RUN     | datm_debug_cfsr
 ###################################################################################################################################################################################
 
 COMPILE | APP=DATM                                                                                                                | - wcoss_cray jet.intel                  | fv3 |
+
+###################################################################################################################################################################################
+# ATM-WAV tests                                                                                                                                                                   #
+###################################################################################################################################################################################
+
+# WW3 not working on Cheyenne in the past, need to check if it works now
```

Comment:
This note re: Cheyenne is no longer applicable?

Reply:
Seems like S2SW is running on cheyenne at least. The next PR that needs new baselines from you/me/... could try to run ATMW on cheyenne in `rt.conf` and remove the note if it works.

Reply:
I should have noticed this earlier. I'll test on Cheyenne and add it to an upcoming PR.

Reply:
Thank you @DeniseWorthen !

Reply:
Checking @JessicaMeixner-NOAA PR #544, she has the ATMW wave tests turned on for Cheyenne.